commands/object-encoding.md

```diff
@@ -3,11 +3,13 @@ Returns the internal encoding for the Redis object stored at `<key>`
 Redis objects can be encoded in different ways:
 
 * Strings can be encoded as `raw` (normal string encoding) or `int` (strings representing integers in a 64 bit signed interval are encoded in this way in order to save space).
+An embedded string, `embstr`, is an object where the internal simple dynamic string, `sds`, is an unmodifiable string allocated in the same chuck as the object itself. `embstr` can be strings with lengths up to the hardcoded limit of `OBJ_ENCODING_EMBSTR_SIZE_LIMIT` or `44` bytes. 
```

Comment:
Add indentation so it follows under the bullet.

Add linebreak between sentences. (See https://github.com/redis/redis-doc#styling-guidelines)

```suggestion
  An embedded string, `embstr`, is an object where the internal simple dynamic string, `sds`, is an unmodifiable string allocated in the same chuck as the object itself.
  `embstr` can be strings with lengths up to the hardcoded limit of `OBJ_ENCODING_EMBSTR_SIZE_LIMIT` or `44` bytes.
```

Btw, you don't think a sublist is necessary? I was thinking something like the below, but your current text is totally fine too.

> * Strings can be encoded as
>   * `raw`, the normal string encoding
>   * `int`, strings representing integers in a 64 bit signed interval are encoded in this way in order to save space
>   * `embstr`, embedded string, an object where.....

Reply:
yes, thanks. i was rushing and forgot to apply this bit of very good feedback.